.travis.yml

```diff
@@ -1,22 +1,3 @@
 language: node_js
```

Comment:
don't we want auto publishing to happen still? i guess buildkite is out of scope for this, but the file seemed fine before outside of the node version? cc @serhalp